### DISCUSSION TITLE
no reaction after press Gernarate Email

### DISCUSSION THREAD

**Discussion by mikedong123** [GorvGoyl/ChatGPT-Writer#50] [Report Bugs and Issues]
there is no reaction after press Gernarate Email /

In china using VPN to use CHATGTP WRITER

https://user-images.githubusercontent.com/124866495/217743427-7eb4b322-e35a-47a1-989e-0eced7e212ab.mp4

**Comment by GorvGoyl** [comment]
which browser are you using? follow these steps for brave: https://chatgptwriter.ai/#brave-issue

**Reply by mikedong123** [reply]
Chrome,....i have to use chrome...

**Reply by friuns2** [reply]
i have same problem on fresh chrome install it works, but on my chrome it don't even i disabled other all plugins.. maybe something todo with chrome://flags

**Reply by mikedong123** [reply]
same as me, i bought a new vps, that works, but for my chrome is not working, already clean the cache. but chapgpt is working...only chat gpt writer not working.

**Reply by friuns2** [reply]
i think its chrome version related, i have  110.0.5481.78 and its not working on fresh account, but on beta or canary it works

**Reply by ZainRain** [reply]
It doesn't work for me either. I am using google chrome atm.

**Reply by friuns2** [reply]
try chrome beta

**Reply by GorvGoyl** [reply]
@mikedong123 @friuns2 @ZainRain could you tell me if this chatgpt extension (https://chatgpt4google.com/) working for you? If that is working then the issue is with my extension else there's some issue with the browser.

**Reply by mikedong123** [reply]
sorry not working, can you please share the chrome bete link, so i can download, thanks a lot.

**Reply by friuns2** [reply]
i use now this older version of chrome where it also works https://www.slimjetbrowser.com/chrome/files/104.0.5112.102/ChromeStandaloneSetup64.exe

**Reply by cesargodoy01** [reply]
Impossible. That extension definitely does not work for my Google Chrome.
